nat/service_iptables.go

```diff
@@ -102,6 +100,6 @@ func (service *serviceIpTables) disableRules() error {
 	return nil
 }
 
-func (service *serviceIpTables) clearStaleRules() error {
-	return service.disableRules()
+func (service *serviceIpTables) clearStaleRules() {
+	service.disableRules()
```

Comment:
If deletion of forwarding rules fails, maybe it's worth at least to add some warning log? I believe that this failure of deletion can mean multiple things, and ignoring that might be dangerous.